s3/replication/manager/src/s3replicationmanager/prepare_job.py

```diff
@@ -0,0 +1,111 @@
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+# prepare_job.py
+#
+# This file contains class which helps to create
+# replication job preparation from s3 metadata
+
+
+import os
+import yaml
+import datetime
+import json
+import uuid
+
+
+class PrepareReplicationJob():
+
+    def __init__(self):
+        """Replication manager's job id."""
+
+        # Job id for replication manager's reference
+        self.rm_job_id = str(uuid.uuid4())
+
+    def prepare_replication_job(self, fdmi_record):
```

Comment:
can be named 
`def from_fdmi(fdmi_record)`
usage 
`replication_job = PrepareReplicationJob.from_fdmi(fdmi_record)`